action.yml

```diff
@@ -103,77 +103,34 @@ outputs:
 runs:
   using: 'composite'
   steps:
-    - name: 'Validate Inputs'
-      id: 'validate_inputs'
+    - name: 'Install pnpm'
```

Comment:
This adds network and I/O overhead to every workflow using this action, which was previously a near-instantaneous shell script. Consider bundling the TypeScript code into a single JavaScript file and committing it to the repository (e.g., dist/validate_inputs.js). This would allow you to run node dist/validate_inputs.js directly without needing to install dependencies at runtime.